code/game/objects/buckling.dm

```diff
@@ -27,6 +27,9 @@
 		if(user_buckle_mob(M, user))
 			return 1
 
+/atom/movable/New()
+	..()
+	buckled_mobs = list()
```

Comment:
This is the point where i realized this optimization is pretty pointless. It had one in lightning already anyway but that file is pretty self-contained. Any idea if the cost of a ..() cloned call is same or lower than new proc by the way ?